js/pages/estimation/inputTypes/ComparativeCohortAnalysis/CreateStudyPopulationArgs.js

```diff
@@ -0,0 +1,25 @@
+define(function (require, exports) {
+
+    var ko = require('knockout');
+
+	function CreateStudyPopulationArgs(data) {
```

Comment:
why not create a class for each of the input types to get rid of `self` pattern?

Reply:
In this case, the self reference isn't necessary because there are no functions/behavior being defined in this class (it's all state data attached to the instance).

I'm not sure a class is even necessary here because, per javascript implementation of classes, classes just manipulate the prototype chain.  I think returning a constructor function isn't a terrible practice since it is clear that without the class directive, there's no behavior being 'templates' to the instance. 

If the idea behind ES6 classes is: 'use them always, don't event think about it', then we should make that clear, but this code is written to not write something 'just for the sake of writing something'.